srcs/utils/isspace.cpp

```diff
@@ -0,0 +1,14 @@
+#include "utils.hpp"
+
+namespace utils {
+
+bool IsSpace(char c) {
+	const char CR = '\r';
+	const char LF = '\n';
+
+	if (c == ' ' || c == '\t' || c == CR || c == LF)
+		return true;
+	return false;
+}
```

Comment:
細かいですがこの関数内で `CR`, `LF` だけ変数にする必要ないかなと思うので、統一してするなら全部変数 or しないなら全部ベタ書き で良さそうです
```suggestion
bool IsSpace(char c) {
	if (c == ' ' || c == '\t' || c == '\r' || c == '\n')
		return true;
	return false;
}
```